### DISCUSSION TITLE
how to set up to write log files everyday?

### DISCUSSION THREAD

**Discussion by magic-thomas** [sveltejs/kit#8700] [Q&A]
Hi. 

I am wondering if sveltekit write log files on local disk with daily, hourly rotating file name. 

I would like to write user ip ,  parameters, and user-agent, referer , etc. 

I just guess it should be on vite. 

Can someone help me?

**Comment by NormandoHall** [comment]
I don't think so. Maybe you want to add a logger like [Winston](https://www.npmjs.com/package/winston)

**Reply by magic-thomas** [reply]
thanks for giving a tip. 

I will study .

thanks.

**Comment by Damian96** [comment]
Hello, I am trying to do the same.
Do we still have to use Winston, or has something else been already added to Svelte Kit?

**Comment by miohtama** [comment]
We are using [Caddy](https://caddy.community/) as a reverse proxy at the front of SvelteKit node.js adapter and log all requests with Caddy JSON format.

[You can find the configuration here](https://github.com/tradingstrategy-ai/proxy-server/blob/7918f3fd0dd4726114e07ef4feb85bceb21b00e5/Caddyfile#L80).
